keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2180,11 +2066,39 @@ message NodePrepareResourcesResponse {
     // will be called again for those that are missing.
     map<string, NodePrepareResourceResponse> claims = 1;
 }
+
+message NodePrepareResourceResponse {
+    // These are the additional devices that kubelet must
+    // make available via the container runtime. A claim
+    // may have zero or more requests and each request
+    // may have zero or more devices.
+    repeated Device cdi_devices = 1 [(gogoproto.customname) = "CDIDevices"];
+    // If non-empty, preparing the ResourceClaim failed.
+    // cdi_devices is ignored in that case.
+    string error = 2;
+}
+
+message Device {
+    // A single device instance may map to several CDI device IDs.
+    // None is also valid.
+    repeated string cdi_ids = 1;
```

Comment:
I would move this to the end for aesthetic reasons.

Reply:
Changed.